frontend/Makefile

```diff
@@ -25,11 +25,11 @@ test: test_unit lint mypy
 
 .PHONY: image
 image:
-	docker build -f public.Dockerfile -t ${IMAGE}:latest .
+	cd .. && docker build -f frontend.public.Dockerfile -t ${IMAGE}:latest . && cd frontend
```

Comment:
Shouldn't this file refer to Dockerfile.frontend.public ?

Reply:
it should, nice catch!